docs/building.md

```diff
@@ -23,3 +23,78 @@ ctest -LE performance
 # run performance tests
 ctest -L performance
 ```
+
+## LANL Darwin - Power 9
+
+### GCC 7.4.0 + Cuda 10.1
+
+The following works on the Darwin Power 9 nodes. We use Spack to install `hdf5` with the `xl`
+compiler. Setting up Spack is beyond the scop of this documentation.
+
+```bash
+# HDF5 is optional, but recommended
+spack install hdf5%gcc@7.4.0
+
+# load required modules and set environment
+module purge
+
+module load cmake/3.17.0
+module load clang/8.0.1
+module load openmpi/p9/4.0.2-gcc_7.4.0
+module load gcc/7.4.0
+module load cuda/10.1
+
+spack load hdf5%gcc@7.4.0
+
+export NVCC_WRAPPER_DEFAULT_COMPILER=g++
+
+# clone parthenon with submodules
+git clone --recursive https://github.com/lanl/parthenon.git
+cd parthenon
+
+# make a build directory
+mkdir build
+cd build
+
+# configure and build (if hdf5 is not available, add -DDISABLE_HDF5=ON)
+cmake -DCMAKE_CXX_COMPILER=$PWD/../external/Kokkos/bin/nvcc_wrapper -DKokkos_ENABLE_CUDA=ON -DKokkos_ENABLE_OPENMP=ON -DKokkos_ARCH_POWER9=ON -DKokkos_ARCH_VOLTA70=ON ..
+make -j
+make test
+```
```

Comment:
I get several compiler errors of the form:

```
driver_types.h:1037:5: error: ‘size_t’ does not name a type
     size_t  pitch;    /**< Pitch of allocated memory in bytes */
```

When trying to compile with these options.

Reply:
Finally, got this to build, though it was with clang.

export NVCC_WRAPPER_DEFAULT_COMPILER=clang++

Reply:
I also have added the cmake flag -DCMAKE_CXX_FLAGS="-Xcudafe --diag_suppress=esa_on_defaulted_function_ignored", there are a ton of warning that are output by the nvcc compiler. Looking over discussions elsewhere they appear to be false positives. They are of the form:

```
warning: __host__ annotation is ignored on a 
warning: __device__ annotation is ignored on a
```

Reply:
Turns out there is also a problem with using cmake 3.17 earlier versions will work such as 3.11